### DISCUSSION TITLE
Does Prisma support the PostgreSQL Time field?

### DISCUSSION THREAD

**Discussion by isaacfink** [prisma/prisma#19102] [Q&A] [answered]
Does Prisma support the PostgreSQL Time or Date field? I didn't find anything in the documentation and it seems like I can only achieve the correct result through extending with js (which is probably gonna make everything super slow

**Comment by nurul3101** [comment, marked as answer]
Hello @isaacfink 👋 

Yes, prisma does support PostgreSQL's native date and time fields.
Here's a [reference](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#postgresql-6) on how you can refer native database types in the schema file.

<img width="709" alt="Screenshot 2023-05-08 at 11 34 21 AM" src="https://user-images.githubusercontent.com/13049676/236746278-40db685b-21d7-45c6-b7dc-0538bfb46afd.png">

**Comment by isaacfink** [comment]
@nurul3101 I went over the native types but missed it and I didn't get any autocomplete in my editor, thanks for answering

**Reply by nurul3101** [reply]
No worries! :)
